### DISCUSSION TITLE
每个词条的讨论页

### DISCUSSION THREAD

**Discussion by ztl8702** [MindongLab/yngdieng#12] [产品功能 ideas]

**Comment by Guanchishan** [comment]
讨论页有打算如何实现吗？要像汉典那样结合入论坛功能还是搞社会化评论框抑或？

**Comment by ztl8702** [comment]
没有具体的计划。取决于我们通过词条的讨论想获得什么。对词条的讨论可能有多种目的：
1. 提出修订（纠错）
1. 用户贡献内容 （UGC）
    * 贡献数据（如发音，释义），填补官方数据的不足
1. 用户之间互相切磋，发表感想

也可以有多种形式：
* 论坛/留言式
* 维基式（即每个页面有一个人人可以编辑的区域）

从简到难大约是：
* 单向的反馈/举报功能 #13 （用户单向反馈，由开发者/词库编辑来修订。如有需要针对某个词讨论可能链接到Github或其他论坛系统）
* 社会化评论（即外部工具，网页嵌入。我们自己不保留数据，但同时也不能定制，对用户的留言能做的也有限）
* 自己实现的wiki
* 自己实现的评论区 （可定制度最高。但要自己实现很多东西。）

目前而言，我认为单向反馈就能提供1、 2的功能。可另设论坛/QQ群，供有需要的人主动发帖、发消息讨论，指出问题。

**Comment by ztl8702** [comment]
@Guanchishan  看到你博客用 https://valine.js.org/ ，感觉不错。墙内加载正常吗？？

**Comment by Guanchishan** [comment]
正常。因为他依托leancloud，这是一款中国人开发的服务。我用leancloud的海外版，墙内也正常访问。

Radium <notifications@github.com> 于2020年5月30日周六 下午8:16写道：

**Comment by Guanchishan** [comment]
> @Guanchishan 看到你博客用 https://valine.js.org/ ，感觉不错。墙内加载正常吗？？

要注意如果不想交钱就不能使用他的文件存储功能，即不要往上面上载文件。我不小心存了点文件，产生了一分钱的消费，又因为用的是海外版只支持海外支付手段，现在无力支付只好等着被销号了。无语。

**Comment by ztl8702** [comment]
我不介意支付，现在的服务器也是要付钱的。而且因为受众在国内长期而言一定要把服务器搬到国内。我正研究备案事宜。

**Comment by insualk** [comment]
> > @Guanchishan 看到你博客用 https://valine.js.org/ ，感觉不错。墙内加载正常吗？？
> 
> 要注意如果不想交钱就不能使用他的文件存储功能，即不要往上面上载文件。我不小心存了点文件，产生了一分钱的消费，又因为用的是海外版只支持海外支付手段，现在无力支付只好等着被销号了。无语。

插一句，我现在人在海外，如果有办法让他人代付的话我应该可以帮忙。

**Comment by Guanchishan** [comment]
> 插一句，我现在人在海外，如果有办法让他人代付的话我应该可以帮忙。

我方才找到愿意帮忙的朋友了。多谢你的好意！

**Comment by Guanchishan** [comment]
2020年8月15日榕典、输入法例会上，子善认为倾向于不要讨论页，参考汉典虽然设计了讨论页，却应者寥寥，极其尴尬。取而代之地，拟在点击词条编辑按钮后设置一个二级页面，用户可以直接动手编辑，也可以留下大概的修改意见以反馈给后台。

**Comment by Guanchishan** [comment]
https://github.com/MindongLab/bangua/issues/3 也许用得到这个评论个功能？

如果榕典现有词条不用评论区的话，我就把这个issue转移到bangua这个repo？

@ztl8702

**Comment by ztl8702** [comment]
留着吧。如果要实现的话也是在这个repo实现。bangua repo可以关掉了。

**Comment by ztl8702** [comment]
我现在倾向于精简。单repo主义。

**Comment by Guanchishan** [comment]
not bad，一切为了实用——只要注意力用得过来。bangua repo的历史issue与时间戳就留着给以后为该功能立传的时候做史料用吧。
btw宁德朋友们最近没回复。
